frontend/src/api/k8s/notebooks.ts

```diff
@@ -490,3 +490,28 @@ export const removeNotebookSecret = (
       })
       .catch(reject);
   });
+
+export const restartNotebook = (
+  notebookName: string,
+  namespace: string,
+  opts?: K8sAPIOptions,
+): Promise<NotebookKind> => {
+  const patches: Patch[] = [
+    {
+      op: 'add',
+      path: '/metadata/annotations/notebooks.opendatahub.io~1notebook-restart',
```

Comment:
There's this notebook updating (and restarting!) logic in a different place? Could it have been used instead?

https://github.com/opendatahub-io/odh-dashboard/blob/d5938bfbe56de729fa6927691ad06d6aecdeb9b2/frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx#L164-L167

Reply:
Hello @jiridanek! Thanks for your reviews. These 2 places have different logic. The PR is trying to find out all the connected notebooks to a PVC and simply patch a restart annotation to all the notebooks to trigger the restart, and the code you mentioned here is trying to update one notebook in one single network call, it's not only patching the annotations, but also containing many other fields, and update them all together. So I think it's hard to merge these 2 places.